modules/virt-about-cloning.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-creating-vms-by-cloning-pvcs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-about-cloning_{context}"]
+= About cloning
+
+When cloning a data volume, the Containerized Data Importer (CDI) chooses one of these Container Storage Interface (CSI) clone methods:
+
+* CSI volume cloning
+* Smart cloning
+
+Both CSI volume cloning and smart cloning methods are efficient, but they have certain requirements for use. If the requirements are not met, the CDI uses host-assisted cloning. Host-assisted cloning is the slowest and least efficient method of cloning, but it has fewer requirements than either of the other two cloning methods.
+
+[id="csi-volume-cloning"]
+== CSI volume cloning
+
+Container Storage Interface (CSI) cloning uses CSI driver features to more efficiently clone a source data volume.
+
+CSI volume cloning has the following requirements:
+
+* The CSI driver that backs the storage class of the persistent volume claim (PVC) must support volume cloning.
+* For provisioners not recognized by the CDI, the corresponding storage profile must have the `cloneStrategy` set to CSI Volume Cloning.
+* The source and target PVCs must have the same storage class and volume mode.
+* If you create the data volume, you must have permission to create the `datavolumes/source` resource in the source namespace.
+* The source volume must not be in use.
+
+
+[id="smart-cloning"]
+== Smart cloning
+
+When a CSI plugin with snapshot capabilities is available, the CDI creates a PVC from a snapshot, which then allows efficient cloning of additional PVCs.
```

Comment:
Since this is technically a new section, I think I'd spell out all the terms for their first mention again:
```suggestion
When a Container Storage Interface (CSI) plugin with snapshot capabilities is available, the Containerized Data Importer (CDI) creates a persistent volume claim (PVC) from a snapshot, which then allows efficient cloning of additional PVCs.
```

The exception would be if "CSI plugin" is some sort of product name that's only ever referred to that way.